src/otx/algorithms/detection/adapters/mmdet/apis/train.py

```diff
@@ -119,11 +121,26 @@ def train_detector(model, dataset, cfg, distributed=False, validate=False, times
     elif cfg.device == "xpu":
         model = build_dp(model, cfg.device, device_ids=cfg.gpu_ids, enable_autocast=bool(fp16_cfg))
         model.to(f"xpu:{cfg.gpu_ids[0]}")
+    elif cfg.device == "hpu":
+        import habana_frameworks.torch.core as htcore
+        from habana_frameworks.torch.utils.library_loader import load_habana_module
+
+        load_habana_module()
+        os.environ["PT_HPU_LAZY_MODE"] = "1"
```

Comment:
It would be better to move here if possible.
https://github.com/openvinotoolkit/training_extensions/blob/575b4ec0740d1512af8826eb075085a4dbef0640/src/otx/algorithms/common/utils/__init__.py#L61-L62